dandiapi/api/models/asset.py

```diff
@@ -127,6 +127,16 @@ class Status(models.TextChoices):
 
     class Meta:
         ordering = ['created']
+        indexes = [
+            # Other statuses are likely too common to index, but pending assets are continually
+            # polled and being moved through the pipeline.
+            models.Index(
+                fields=['status'],
+                name='%(app_label)s_%(class)s_status_pending',
+                # TODO: refer to Pending via the Status enum
```

Comment:
I'm working on a follow up PR for this. The tldr is we shouldn't define the enums inside of model classes because it otherwise becomes impossible to refer to them inside of meta classes. So `Asset.Status` needs to be moved to a top level `AssetStatus`. FYI @mvandenburgh